Automattic/woocommerce-services
# Fix NUX images path so they get included in the release ZIP

Fixes #1187 

The `release` task was including the `images/` folder in the plugin `zip`, but the images were in the `assets/images` folder instead, so they weren't being included in the release `zip`.

To test:
- Run the release task (`npm run release`)
- Install the resulting plugin in a site.
- Go through any of the NUX cases.
- Check that the images appear.
- Optional: Check that there's a folder in the plugin `zip` called `images`, with the banner images inside it.

PS: I also removed the `tos-notice-bg.png` image because it wasn't being used.
